perun-dispatcher/src/main/java/cz/metacentrum/perun/dispatcher/service/impl/DispatcherManagerImpl.java

```diff
@@ -149,4 +135,10 @@ public void cleanOldTaskResults() {
 		}
 	}
 
+	@Override
+	public void startSchedulingTasks() {
+		Thread taskSchedulingThread = new Thread(taskScheduler);
+		taskSchedulingThread.start();
```

Comment:
The thread itself must have some kind of error handling, at least it should log errors properly. 
Then, we can have some procedure in place for automatic restarts of taskSchedulingThread in case it dies.